popup.js

```diff
@@ -0,0 +1,322 @@
+              the releaLorem Ipsum is simply dummy text of the printing
+              and typesetting industry. Lorem Ipsum has been the
+              industry's standard dummy text ever since the 1500s, when an
+              unknown printer took a galley of type and scrambled it 1960s
+              with the release.`,
+    technologies: ['html', 'css', 'javascript', 'bootstrap', 'ruby', 'github'],
+  },
+];
+
+const projectCards = [
+  {
+    card: 1,
+    name: 'Tonic',
+    name2: 'Tonic',
+    image: 'images/snap.png',
+    image2: 'images/tonic1.png',
+    description: `A daily selection of privately personalized reads; no accounts or
+                              sign-ups required.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['CANOPY', 'Back End Dev', '2015'],
+  },
+  {
+    card: 2,
+    name: 'Multi-Post Stories',
+    name2: 'Multi-Post Stories',
+    image: 'images/story.png',
+    image2: 'images/multi2.png',
+    description: `Experimental content creation feature that allows users to add to
+                              an existing story over the course of a day without spamming their
+                              friends.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['FACEBOOK', 'Full Stack Dev', '2015'],
+  },
+  {
+    card: 3,
+    name: 'Tonic',
+    name2: 'Facebook 360',
+    image: 'images/snap1.png',
+    image2: 'images/portcard.png',
+    description: ` Exploring the future of media in Facebook's first Virtual Reality
+                              app; a place to discover and enjoy 360 photos and videos on Gear
+                              VR.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['FACEBOOK', 'Full Stack Dev', '2015'],
+  },
+  {
+    card: 4,
+    name: 'Multi-Post Stories',
+    name2: 'Uber Navigation',
+    image: 'images/snap3.png',
+    image2: 'images/uber2.png',
+    description: `  A smart assistant to make driving more safe, efficient, and fun by
+                              unlocking your most expensive computer: your car.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['UBER', 'Lead Developer', '2018'],
+  },
+];
+
+const portfolioSection = document.getElementById('portfolio');
+portfolioSection.innerHTML = projectCards
+  .map(
+    (project, index) => `
+  <section class="card ${index % 2 !== 0 ? 'image-left' : 'image-right'}">
+<div class="content-right">
+          <img class="mobileCard" src="${project.image}" alt="Tonic image" />
+          <img
+            class="desktopCard"
+            src="${project.image2}"
+            alt="Nature image"
+          /></div>
+
+          <div class="content-left">
+          <h2 class="mobileheadline">${project.name}</h2>
```

Comment:
- [ ] After implementing the required changes on the spacing, we now have a new problem: the project image is now too big and doesn't follow the template's design guidelines. Please fix it 🙏 
- [ ] Please remove the gray area I highlighted in the attachment below as it is is not in the original design 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/162569105-47680b38-50a9-43df-85da-1c67cb2bc6a6.png)

## Figma Design
![image](https://user-images.githubusercontent.com/46839548/162569157-9ce4c1f4-0a7d-4593-85d0-04b2738f31e2.png)